shepherd

```diff
@@ -5,6 +5,15 @@ server_version() {
   docker version -f "{{.Server.Version}}"
 }
 
+logger() {
+  local log="$1"
+  local is_verbose_log="${2:-false}"
+  
+  if [ "$is_verbose_log" == "true" -a "$verbose" == "true" -o "$is_verbose_log" == "false" ]; then
```

Comment:
I'd rather like to use the (in my opinion better) newer condition syntax with `&&` and `||`.